types/jest-each/jest-each-tests.ts

```diff
@@ -0,0 +1,17 @@
+import * as each from 'jest-each';
+
+const params = [[1, 0, 1], [1, 1, 0], ['1', 'two', 'three']];
+
+each(params).test('', () => {});
+each(params).it('', () => {});
```

Comment:
Just check in the [source code](https://github.com/mattphillips/jest-each/blob/master/src/index.js#L14-L15), seems `it.skip` and `it.only` is also exported. We can add them here as test cases as well: 

```js
each(params).it.skip('', () => {})
each(params).it.only('', () => {})
```

Reply:
Right you are! Can you check this new commit for me?

Reply:
Good to go mate 😀